docs/faq.rst

```diff
@@ -12,6 +12,39 @@ Yes, the `deduplication`_ technique used by
 |project_name| makes sure only the modified parts of the file are stored.
 Also, we have optional simple sparse file support for extract.
 
+If you use tools like Borg (or rsync, tar, cp and most other backup tools) to back up
```

Comment:
most other backup tools is very vague. But i’m not sure what we could say instead that is not tautological like "other backup tools that don‘t use special file system snapshot features of the operating system". (also way to long)